Review and improve legal notice for clarity and accuracy (#291)
organicmaps/organicmaps.github.io: content/terms/index.md

---
title: "Legal information"
description: "Legal notices and licensing details for Organic Maps"
weight: 1000
extra:
  menu_title: Legal
---

### App

Copyright 2014-2021 My.com B.V. (Mail.Ru Group) \
Copyright 2020-2025 Organic Maps (OMaps) Contributors; \
See the full list of contributors [here](https://github.com/organicmaps/organicmaps/graphs/contributors?from=12%2F20%2F2022) at Github.

This app is licensed under the Apache License, Version 2.0 (the "License"); you may not use this application except in compliance with the License.
You may obtain a copy of the Apache License 2.0 at

[http://www.apache.org/licenses/LICENSE-2.0][license]

Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.

### Website

> Copyright 2020-2025 Organic Maps (OMaps) Contributors; \

Reviewer: Wasn't this website and repo created in 2021?
```suggestion
Copyright 2021-2025 Organic Maps (OMaps) Contributors; \
```

---
title: "Legal information"
description: "Legal notices and licensing details for Organic Maps"
weight: 1000
extra:
  menu_title: Legal
---

### App

> Copyright 2010-2021 My.com B.V. (Mail.Ru Group) \

Reviewer: ```suggestion
Copyright 2014-2021 My.com B.V. (Mail.Ru Group) \
```

Copyright 2020-2025 Organic Maps (OMaps) Contributors; \

> See the [CONTRIBUTORS][contributors] file in the [organicmaps/organicmaps][github] repository
> for a full list of contributors.

Reviewer: Ditto.
```suggestion
See the full list of contributors [here](https://github.com/organicmaps/organicmaps/graphs/contributors?from=12%2F20%2F2022) at Github.
```

This app is licensed under the Apache License, Version 2.0 (the "License"); you may not use this application except in compliance with the License.
You may obtain a copy of the Apache License 2.0 at

[http://www.apache.org/licenses/LICENSE-2.0][license]

Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.

### Website

Copyright 2020-2025 Organic Maps (OMaps) Contributors; \

> See the [CONTRIBUTORS][contributors] file in the [organicmaps/organicmaps][github] repository
> for a full list of contributors.

Reviewer: ```suggestion
See the full list of contributors [here at Github](https://github.com/organicmaps/website/graphs/contributors).
```
